### PR TITLE
WIP #Hackathon - Add VJEPA to SmolVLA

### PR DESCRIPTION
**Pull request by burgalon**
This is a WIP for adding V-JEPA2 to SmolVLA

1. Currently we concat the V-JEPA2 in addition to the VLM embeddings which might make it too bulky. We're considering removing the SmolVLM in favor of some language only SigLIP embedding?
2. It's possible that the prepare_images() is currently normalizing the images in a way that does not fit/suit V-JEPA2 - not sure yet if this is indeed the case, and how to fix it
3. Need code to run the full pre-training by retrieving all datasets tagged lerobot. Alternatively, we could consider loading the pretrained base model and expanding the relevant expert layer and starting training from there - not sure if it makes sense

Would love to get any feedback and ideas to improvement

Training with stacking dataset
https://wandb.ai/burgalon/lerobot/runs/83njsme5

Would love your feedback
@mshukor @danaaubakirova @pkooij @fracapuano

### CODE REVIEW

**Comment by fracapuano** [comment]
wow @burgalon so cool! Can't wait to see some results 😉

**Comment by burgalon** [comment]
Our fork contains multiple different trials in trying to concatenate VJEPA in [different](https://github.com/astriaai/lerobot/tree/vjepa-tome-prefix) ways however currently we haven't been able to produce meaningful results. Closing this for now

**Comment by fracapuano** [comment]
Hey @burgalon thank you for this PR. I think it's a very interesting trial---I'd be very happy to collaborate on a blogpost to share your learnings with the community if you feel like it 🙌 VJEPA looks great and it'd be a cool set of experiments. Reach out and we can totally organize this if you're interested! ❤️ 

Also, congrats on trying the idea out---so interesting!
